rosbag2_examples/rosbag2_examples_py/rosbag2_examples_py/compressed_bag_recorder.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2025 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import rclpy
```

Comment:
Please add one empty line for separation between the license text and Python imports

Reply:
Thanks for reviewing! The extra line is included now. :+1: